### DISCUSSION TITLE
Extracting Streamer name data from Azuracast stream

### DISCUSSION THREAD

**Discussion by SpeedyCharly** [AzuraCast/AzuraCast#5286] [Q&A] [answered]
Does anyone have a quick and easy method for polling the server at regular intervals to extract the streamer name (dj name) that is normally shown on the public player during a live cast.  That info would then be passed to a variable to effect some work in our web site (ei; triggering dj's photo to appear on the web page
![Image2](https://user-images.githubusercontent.com/66107794/162896056-26304716-e810-4c28-917d-1068cf02f488.jpg)
)

**Comment by BusterNeece** [comment, marked as answer]
@SpeedyCharly In the [NowPlaying API response](https://demo.azuracast.com/api/nowplaying_static/azuratest_radio.json), it'll be listed under `live`, then `streamer_name`. For non-live streams, it'll be an empty string.

**Reply by SpeedyCharly** [reply]
@SlvrEagle23 Thanks, guess I'll have to knuckle-down and learn me the API process LOL :P

**Reply by Countryman98** [reply]
Me too @SpeedyCharly. Would be great to be able to add a dj/show image into php website pages but the api stuff is above my head!
An "Onair/Live" widget in Azuracast would be the Sh*^
